lib/chitragupta/chitragupta.js

```diff
@@ -73,11 +74,16 @@ function jsonLogFormatter(options) {
   return JSON.stringify(util.sanitizeKeys(options.level, options.message, options.meta));
 }
 
+function setUserIdOnCLS(userId){
+  cls.set('userId',userId);
+}
```

Comment:
@mayank2498 as discussed, we should be making this generic to ensure we can set keys in different scopes.